### PR TITLE
Update generate_cifar10_tfrecords.py

### PR DESCRIPTION
**Pull request by newcoder**
change cPickle to pickle for python3
specify  encoding='latin1' when call pickle.load()
@nealwu

### CODE REVIEW

**Comment by newcoder** [comment]
CLA signed.

**Comment by tfboyd** [comment]
Will give the patch a try today and then accept.

**Comment by tfboyd** [comment]
Testing now using Python2 on Ubuntu TF 1.5ish likely.  A day late.  :-)  

- Execute download
- run processed data on CPU

Looks good approved, I assume author tested Python3.  I should have had some unit tests for this thing.  Next time for sure.  :-)

**Comment by tfboyd** [comment]
Sorry still testing.  Pushed send to fast.

**Comment by tfboyd** [comment]
INFO:tensorflow:Evaluation [100/100]
INFO:tensorflow:Finished evaluation at 2018-02-08-21:39:08
INFO:tensorflow:Saving dict for global step 1000: accuracy = 0.4919, global_step = 1000, loss = 2.47612

Did not have time to run a full test, just wanted to verify the data was accurate.
